src/bool.rs

```diff
@@ -1,12 +1,15 @@
 //! Functions for safe transmutation to `bool`.
+//! 
 //! Transmuting to `bool' is not undefined behavior if the transmuted value is
```

Comment:
This is an apostrophe

![misformat SS](https://user-images.githubusercontent.com/6709544/50738525-454ea900-11d5-11e9-8531-361d1ebe6091.png)